torch/utils/cpp_extension.py

```diff
@@ -1865,7 +1865,8 @@ def load_inline(name,
                 is_python_module=True,
                 with_pytorch_error_handling=True,
                 keep_intermediates=True,
-                use_pch=False):
+                use_pch=False,
+                no_header=False):
```

Comment:
no_extension_header

Reply:
I'm considering following @ngc92 advice and actually have 0 headers so a true no header mode.